content/influxdb/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,31 @@ menu:
 weight: 101
 ---
 
+## v2.0.2 General Availability [2020-11-17]
+
+### Features
+- Improvements to upgrade from 1.x to 2.x:
+    - Warning appears if auth is not enabled in 1.x, which is not an option in 2.x. For details, see [Upgrade from InfluxDB 1.x to InfluxDB 2.0](/influxdb/v2.0/upgrade/v1-to-v2/).
+    - `upgrade` command now checks to see if continuous queries are running and automatically exports them to a local file.
+- Upgrade to [Flux v0.95.0](/influxdb/v2.0/reference/release-notes/flux/#v0-95-0-2020-11-17).
+- Manage database retention policy (DBRP) mappings via CLI. See [`influx v1 dbrp`](/influxdb/v2.0/reference/cli/influx/v1/dbrp/)
+
+### Bug Fixes
+- Fixes to `influx upgrade` command:
+  - Remove internal subcommands from help text.
+  - Validate input paths upfront.
+- Add locking during TSI iteration creation.
+- Fix various typos.
+- Use `--skip-verify` flag for backup/restore CLI command.
```

Comment:
... CLI command. This is passed to the underlying HTTP client for the `BackupService` and `RestoreService` to support backup and restore on servers with self-signed certificates.